module/VuFind/src/VuFind/Controller/MyResearchController.php

```diff
@@ -2235,6 +2308,16 @@ public function deleteLoginTokenAction()
         }
         $series = $this->params()->fromPost('series', '');
         $this->getAuthManager()->deleteToken($series);
+
+        $this->getAuditEventService()->addEvent(
+            AuditEventType::User,
+            AuditEventSubtype::DeleteLoginToken,
+            $this->getUser(),
+            data: [
+                'series' => $series,
+            ]
```

Comment:
Could use a `compact` here to save a couple of lines.

```suggestion
            data: compact('series')
```